composer/profiler/profiler_schedule.py

```diff
@@ -12,10 +12,10 @@
 
 
 def cyclic_schedule(
-    skip_first: int = 0,
+    skip_first: int = 1,
     wait: int = 0,
     warmup: int = 1,
-    active: int = 4,
+    active: int = 50,
```

Comment:
CC @bandish-shah  on these changes -- they look fine to me.

Reply:
This is probably fine, the one issue I can think of is if someone enables the Torch profiler.  The wall clock time is significant when we turn it on so if a user runs with the defaults and enables the Torch profiler, it could slow down the workload substantially.  Is there a smaller value, maybe 10-20 that would work?  Alternatively could we have a schedule per trace handler vs a global schedule?  My guess is that would require a significant amount of re-factor to the profiler to support.